<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="TabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="Book">
       <attribute name="title">
        <string>Book</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="tableViewBook"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAddBook">
          <property name="text">
           <string>AddBook</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Author">
       <attribute name="title">
        <string>Author</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tableViewAuthor"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAutor">
          <property name="text">
           <string>Add Author</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Publisher">
       <attribute name="title">
        <string>Publisher</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="tableViewPublisher"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonPublisher">
          <property name="text">
           <string>Add Publisher</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Style">
       <attribute name="title">
        <string>Style</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="tableViewStyle"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStyle">
          <property name="text">
           <string>Add Style</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Order">
       <attribute name="title">
        <string>Order</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTableView" name="tableViewOrder"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonOrder">
          <property name="text">
           <string>Add Order</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ShopList">
       <attribute name="title">
        <string>OrderList</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableView" name="tableViewOredrBooks"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonOrderList">
          <property name="text">
           <string>Add ShopList</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
